gen3-client/jwt/functions.go

```diff
@@ -296,12 +296,17 @@ func (f *Functions) CheckPrivileges(profile string, configFileType string) (stri
 		return "", nil, errors.New("Error occurred when unmarshalling response: " + err.Error())
 	}
 
-	projectAccess, ok := data["project_access"].(map[string]interface{})
-	if !ok {
-		return "", nil, errors.New("Not possible to read user access privileges")
+	resourceAccess, ok := data["authz"].(map[string]interface{})
+
+	// If the `authz` section (Arborist permissions) is empty or missing, try get `project_access` section (Fence permissions)
+	if len(resourceAccess) == 0 || !ok {
```

Comment:
Q for @paulineribeyre maybe: is this fallbeck check vaild? could there be user with empty `authz` section (which is intended), but no `project_access` section?

Reply:
looks fine to me, `project_access` is always included